internal/envoy/auth/parser.go

```diff
@@ -46,15 +46,14 @@ type CloudEntityBuilderArguments struct {
 	Method    string
 }
 
-type parseAuthOptionsArguments struct {
+type ParseAuthArguments struct {
 	Logger                       logr.Logger
 	EnvoyConfiguration           *config.EnvoyConfiguration
 	HTTPConnectionManagerBuilder *config.HCMBuilder
 	CloudEntityBuilderArguments  *CloudEntityBuilderArguments
 	CloudEntityBuilder           *cloudentity.Builder
 	GenerateClusterName          generateClusterNameFunc
 	KubernetesClient             client.Client
-	PathsToIgnore                []string
 }
 
 func NewParseAuthOptionsArguments(
```

Comment:
this has become pointless as we can achieve the same by initializing the variable instead of using constructor

Reply:
Resolved.

Removed `NewParseAuthOptionsArguments` constructor.